help/md/dnDirichlet.md

```diff
@@ -3,9 +3,9 @@ dnDirichlet
 ## title
 Dirichlet Distribution
 ## description
-Dirichlet probability distribution on a simplex.
+The Dirichlet probability distribution is a generalization of the Beta distribution for multiple variables.
 ## details
-The Dirichlet probability distribution is the generalization of the Beta distribution. A random variable from a Dirichlet distribution is a simplex, i.e., a vector of probabilities that sum to 1.
+The Dirichlet distribution takes one parameter, alpha, a vector of numbers representing the concentration of the distribution on each variable. It then returns a simplex (i.e. a vector whose elements sum to 1) representing the relative probability of each variable. 
```

Comment:
We could also add that when all the alpha's are 1, then it is uniform on the simplex.